substrate/frame/revive/src/evm/runtime.rs

```diff
@@ -693,9 +692,8 @@ mod test {
 					log::debug!(target: LOG_TARGET, "Gas price: {:?}", tx.gas_price);
 				}))
 				.unwrap();
-
-		let expected_tip =
-			tx.gas_price.unwrap() * tx.gas.unwrap() - U256::from(GAS_PRICE) * tx.gas.unwrap();
-		assert_eq!(U256::from(extra.1.tip()), expected_tip);
+		let diff = tx.gas_price.unwrap() - U256::from(GAS_PRICE);
```

Comment:
Please no unchecked math in runtime. Let's be paranoid here.

Reply:
that's a test though should be fine :upside_down_face: